src/Messages/OWSMessageSender.m

```diff
@@ -471,8 +480,15 @@ - (void)resendMessageFromKeyError:(TSInvalidIdentityKeySendingErrorMessage *)err
 
         // Normally marking as unsent is handled in sendMessage happy path, but beacuse we're skipping the common entry
         // point to message sending in order to send to a single recipient, we have to handle it ourselves.
-        void (^markAndFailureHandler)(NSError *error) = ^(NSError *error) {
+        RetryableFailureHandler markAndFailureHandler = ^(NSError *error, BOOL isRetryable) {
             [self saveMessage:message withError:error];
+            if (isRetryable) {
+                DDLogWarn(@"Skipping retry for message failure in this corner case.");
+                // Fixing this will require a larger refactor. In the meanwhile, we don't
+                // retry message sending from accepting key-changes in a group. (Except for the inner retry logic w/ the
+                // messages API)
```

Comment:
So at the moment we're only logging, but not changing the retry behavior here?

I tend to mark comments like this with a standard prefix: TODO: for readability and to make them grep-able later.